src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx

```diff
@@ -134,33 +139,52 @@ function Service({
           updateChangeSource(data, formik)
           hideDrawer()
         }
+        if (isTemplate && formik.dirty) {
+          updateTemplate?.({
+            identifier,
+            name: 'Test Template',
+            orgIdentifier,
+            projectIdentifier,
+            spec: {
+              spec: { serviceRef: serviceRef, environmentRef: environmentRef },
+              timeout: '2m',
+              type: 'Http'
+            } as any,
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9235241)
```suggestion
            } as unknown,
```